frontend/web/components/pages/AnimeDetails.js

```diff
@@ -28,25 +29,56 @@ class AnimeDetails extends React.Component {
   render () {
     const { animeCache } = this.props;
     let content = null;
-    let details = animeCache[this.props.params.id];
+    let details = animeCache[this.props.params.id]
     if (details) {
-      console.log('got details');
-      let averageScore = (Math.round(parseInt(details.average_score, 10)))/10;
-      let description = details.description.replace(/<[^>]*>/ig, "");
+      console.log('got details')
+      console.log(details);
+      let averageScore = (Math.round(parseInt(details.average_score, 10)))/10
+      let description = details.description.replace(/<[^>]*>/ig, "")
+      let characters = []
```

Comment:
you can declare this without the empty array, since you're doing functional programming and  `slice returns a new array`. You're basically declaration an empty an array and then making a new array and overwriting the old array. You may need to do this if you were passing this down to a child component and that component was mapping over it, but in this instance doesn't look like it's necessary